lib/matplotlib/backend_bases.py

```diff
@@ -604,6 +590,9 @@ def get_text_width_height_descent(self, s, prop, ismath):
 
         Whitespace at the start and the end of *s* is included in the reported width.
         """
+        # Note: Using the ``bbox`` property of Text objects, backend
```

Comment:
I'm unclear how this relates to the context.

What exactly do backend implementors implement? Do they reimplement this function?
Also, how do they check? Do they draw a text and additionally its bbox? Would a minimal example be helpful?

Reply:
IMHO the comment is not useful as is. Either you already know the context and then likely don't need it, or it's too terse and you still don't know how to apply it. I'm fine going either way, removing or expanding.

Reply:
Let's just kill it, then.